src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1633,27 +1633,19 @@ private RefreshItem getRefreshToken(final AuthenticationRequest request) {
             if (item == null || StringExtensions.IsNullOrBlank(item.getRefreshToken())) {
                 // if not present, check multiResource item in cache. Cache key
                 // will not include resourceId in the cache key string.
-                Logger.v(TAG, "Looking for Multi Resource Refresh token");
+                Logger.v(TAG + methodName, "Looking for Multi Resource Refresh token");
                 keyUsed = CacheKey.createMultiResourceRefreshTokenKey(request, userId);
                 item = mTokenCacheStore.getItem(keyUsed);
                 multiResource = true;
             }
             
-            // If still cannot find an item, try with family clientid
```

Comment:
I just want to have it temporarily ripped out. When server returns a single FoCI RT, it will make the look up logic way clearer I think.